applications/serial_lte_modem/doc/GNSS_AT_commands.rst

```diff
@@ -7,54 +7,103 @@ GNSS AT commands
    :local:
    :depth: 2
 
-The following commands list contains GNSS-related AT commands.
+The following list contains GNSS-related AT commands.
 
-GNSS
-====
+Control GNSS
+============
 
 The ``#XGPS`` command controls the GNSS.
 
 Set command
 -----------
 
-The set command allows you to start and stop the GNSS.
+The set command allows you to start and stop the GNSS module.
+
+Both the :ref:`lib_nrf_cloud_agps` service and the :ref:`lib_nrf_cloud_pgps` service can be used with the module, either together or independently of each other.
+Using them reduces the time it takes the GNSS module to estimate its position.
+
+To use either of them, the device must be connected to nRF Cloud (using the :ref:`#XNRFCLOUD <SLM_AT_NRFCLOUD>` AT command) when starting the GNSS module.
+In addition, the following Kconfig options must be enabled:
+
+   * :kconfig:option:`CONFIG_NRF_CLOUD_AGPS <CONFIG_NRF_CLOUD_AGPS>` to use A-GPS.
+   * :kconfig:option:`CONFIG_NRF_CLOUD_PGPS <CONFIG_NRF_CLOUD_PGPS>` to use P-GPS.
+
+When using either assistive service, the choice of which are used is made at compile time only.
```

Comment:
Would this work:
```suggestion
Assistive services cannot be enabled after compile time.
```
Alternatively, if the Kconfig options are what must be enabled at compile time, then it makes sense to move this information to the sentence that introduces them, so using this for Line 26 and removing Line 31:
```
In addition, the following Kconfig options must be enabled at compile time:
```

Reply:
Hmm, in that case, I might move this to a negative declaration like so:
```suggestion
If both assistive services are enabled at compile time, you cannot choose to use only one of them at run time.
```